sys-auth/ssh-ldap-pubkey/ssh-ldap-pubkey-1.3.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+inherit distutils-r1
+
+DESCRIPTION="Utility to manage SSH public keys stored in LDAP"
+HOMEPAGE="https://github.com/jirutka/ssh-ldap-pubkey"
+
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/jirutka/${PN}/${PN}.git"
+
+	inherit git-r3
+else
+	SRC_URI="https://github.com/jirutka/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+MY_CDEPEND="
```

Comment:
Why don't you name this CDEPEND or COMMON_DEPEND like everyone else?

Reply:
My argument would be, that if I **should not** use *CDEPEND* because a future EAPI **could** make use of it, same argument applies to any other own variable in global scope, not? That's why I decided to prefix with *MY_* to be sure to never ever clash with any upcoming EAPI.

But I can rename to *COMMON_DEPEND* if it makes you happy.